### PR TITLE
Documentation Updates

### PR DESCRIPTION
**Pull request by zitterbewegung**
I made some documentation updates that would be helpful.

### CODE REVIEW

File: docs/devguide/installation.rst [hunk 1]
```diff
@@ -44,6 +44,13 @@ Windows
 OS X
 ~~~~
 
+After cloning the repository then install yotta using the app and launch the app. You can find the latest release of yotta for OS X at https://github.com/ARMmbed/yotta_osx_installer/releases/ .
+
+When you are at the yotta terminal execute the following commands. 
+>>> yt target bbc-microbit-classic-gcc-nosd
+>>> yt up
+>>> yt make
```
Comment (dpgeorge) [inline docs/devguide/installation.rst]: This is already stated in the main README.md file in the root directory.

File: docs/devguide/installation.rst [hunk 2]
```diff
@@ -44,6 +44,13 @@ Windows
...
+You should have a firmware.hex file that you can now upload to your microbit.
```
Comment (dpgeorge) [inline docs/devguide/installation.rst]: Where is the firmware.hex file?  Maybe simpler to just say: "Now follow the instructions in README.md".

**Comment by ntoll** [comment]
@zitterbewegung care to update the PR with a reference to the README..? If you rebase and squash into a single commit against master then I'll merge. :-)

**Comment by zitterbewegung** [comment]
@ntoll Ok I have rebased and squashed into a single commit.

**Comment by zitterbewegung** [comment]
@ntoll Hi I believe I rebased and squashed into a single commit. I accidentally added another documentation update but I reverted that change.

**Comment by dpgeorge** [comment]
@zitterbewegung nope, this is still 8 commits :(

**Comment by willingc** [comment]
@zitterbewegung [Here](https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit)'s a wiki entry from another project that walks through rebase and squash.

**Comment by dpgeorge** [comment]
Rebased and merged in 5322658d7db42cb552408becc6f9340fdab44f27, without the "yt" parts, to match other sections in that file.

**Pull request closed without merging**